README.md

```diff
@@ -1,2 +1,57 @@
 # freeradius-datadog-metrics
-Datadog custom check for FreeRADIUS
+
+This repo contains a Datadog agent check to monitor [FreeRADIUS](https://freeradius.org/). The check will publish one service check to Datadog, called `freeradius`. It also records a number of custom metrics about what the FreeRADIUS process is doing.
+
+
+## Installation
+
+1. [Configure a status server](https://wiki.freeradius.org/config/Status) on your FreeRADIUS installation.
```

Comment:
This uses `radclient` -- should we call out that they may have to install the right package so it's available? alternatively this can be re-written to use `pyrad`, but i dont think having a python dependency is any better.

Reply:
Good point, `README.md` updated. It sounds a little difficult to install Python dependencies in the Datadog environment, which is one reason I went with `radclient`. This is also what `collectd-freeradius` does, which is the code I modeled this on.